lightning-invoice/src/payment.rs

```diff
@@ -740,6 +740,28 @@ where
 	}
 }
 
+/// Data structure used to track information about inflight HTLCs.
+pub struct InflightHtlcs {
```

Comment:
Let's capitalize the "f" for consistency with with `AccountsForInFlightHtlcs`. It "looks right" given in-flight is hyphenated. Though, I do prefer to keep variable names as `inflight` for no other reason than that it looks better than `in_flight`. 😛

Reply:
good point 😅

Reply:
resolved in [d953a05](https://github.com/lightningdevkit/rust-lightning/pull/1694/commits/d953a059d5ea25296a9e0e7f7abe65480e0d14bd)